### DISCUSSION TITLE
How to deploy an angular application with environment.ts

### DISCUSSION THREAD

**Discussion by diegoberges** [vercel/vercel#7680] [Help]
Hello,

I am trying to implement an application in angular that uses the environment.ts file and the compilation gives an error because it cannot retrieve the environment variables (the file is not uploaded to GITHUB because there are private keys).

How could I deploy to vercel?

Thank you

**Comment by KarkatSG** [comment]
Have this same issue. Bump.

**Reply by mehul1011** [reply]
This issue is quite critical to be looked into, mostly all angular app needs API keys stored in the environment file.
